client/src/Account/components/Account.tsx

```diff
@@ -13,14 +13,17 @@ import { Spinner } from 'common/components'
 // layout
 import { NotFound } from 'layout/components'
 import useMediaQuery from 'common/hooks/useMediaQuery'
+import { accountConverter } from 'common/helpers/accountConverter'
 
 const Account: FC = () => {
   const { accountId } = useParams()
 
+  const address = accountConverter(accountId || '')
```

Comment:
Hm... is it okay for `accountId` to not exist? It is also a strange API for converter to silently return empty string on invalid input, I'd expect it to throw an error rather than eating it internally.

If this context allows for empty strings for some reason, that it should be something like this instead:
```typescript
const address = accountId ? accountConverter(accountId) : ''
```